web/rootfs/etc/cont-init.d/10-config

```diff
@@ -97,10 +97,16 @@ if [[ ! -f /config/config.js ]]; then
             /config/config.js
     fi
 
-    if [[ ! -z "${ETHERPAD_URL_BASE}" && -z "$(grep -om1 'etherpad_base:' /config/config.js)" ]]; then
-        sed -i \
-            -e "/enableWelcomePage/a\    etherpad_base: '${PUBLIC_URL}/etherpad/p/'," \
```

Comment:
Can you please remove the /etherpad/ proxying in nginx in the case of ETHERPAD_PUBLIC_URL being specified?

Reply:
@saghul Not sure to understand... I mean, the code you pointed out is the one that's removed, not the new code 😅 

In case we use the `ETHERPAD_PUBLIC_URL`, we already do not use the nginx proxying:
```shell
        if [[ ! -z "${ETHERPAD_PUBLIC_URL}" ]]; then
            sed -i \
                -e "/enableWelcomePage/a\    etherpad_base: '${ETHERPAD_PUBLIC_URL}/p/'," \
                /config/config.js
        elif [[ ! -z "${ETHERPAD_URL_BASE}" ]]; then
```

If I misunderstood, can you make a code suggestion ?

Reply:
I mean in the Nginx config sorry :-) Ah wait, it might not be needed after all! Nevermind...

Reply:
I confirm that it's not needed since it's already done.

The nginx proxying for Etherpad is only triggered for `ETHERPAD_URL_BASE`, not for `ETHERPAD_PUBLIC_URL`:
https://github.com/jitsi/docker-jitsi-meet/blob/a7563d4f82d79cfaca6f5ff408299f7857f7cac7/web/rootfs/defaults/meet.conf#L48-L52

So if Jitsi is properly configured (ie. only `ETHERPAD_PUBLIC_URL` to external Etherpad and `ETHERPAD_URL_BASE` left empty), the proxying will not be generated.